src/lib/arch/aarch64/InstructionMetadata.cc

```diff
@@ -2000,6 +2083,18 @@ void InstructionMetadata::revertAliasing() {
       }
       return aliasNYI();
     case ARM64_INS_REV64:
+      // rev64 vd.t, vn.t
+      if (opcode == Opcode::AArch64_REV64v16i8 ||
+          opcode == Opcode::AArch64_REV64v2i32 ||
+          opcode == Opcode::AArch64_REV64v4i16 ||
+          opcode == Opcode::AArch64_REV64v4i32 ||
+          opcode == Opcode::AArch64_REV64v8i16 ||
+          opcode == Opcode::AArch64_REV64v8i8) {
+        operandCount = 2;
+        operands[0].access = CS_AC_WRITE;
+        operands[1].access = CS_AC_READ;
+        return;
+      }
```

Comment:
I am confused by this alias addition. If the access rights are not correct then add it to the main switch statement instead of the revertalias function.

If they do infact alias to something weird and this is required, then please make this clear in the comment